src/components/Common/DropdownMenu.tsx

```diff
@@ -300,7 +300,7 @@ export const MenuItem = forwardRef<
         role="menuitem"
         className={classNames(
           menuItemClassNames,
-          'h-[42px] w-full px-3',
+          'h-[34px] w-full px-3',
```

Comment:
You need to fix on line 212 also to fix it:
![image](https://github.com/epam/ai-dial-chat/assets/20163971/505daa02-5e69-4047-89b4-ed628bbd1e0d)